src/transport/net.cc

```diff
@@ -1317,24 +1385,33 @@ static ncclResult_t recvProxyProgress(struct ncclProxyState* proxyState, struct
         struct ncclProxySubArgs* sub = subGroup + i;
```

Comment:
The empty line before this for loop is removed in nccl 2.23 as well, we need remove it to avoid non-MSCCL changes

Reply:
Saw the following diff when comparing NCCL 2.23 and this PR
```
       int sizes[NCCL_PROXY_MAX_SUBS];
       int tags[NCCL_PROXY_MAX_SUBS];
       void* mhandles[NCCL_PROXY_MAX_SUBS];
+
       for (int i=0; i<subGroup->groupSize; i++) {
         struct ncclProxySubArgs* sub = subGroup + i;
         if (sub->posted < sub->nsteps) {
```

Reply:
Resolved in latest commit.